adafruit_mpu6050.py

```diff
@@ -347,3 +372,17 @@ def cycle_rate(self, value: int) -> None:
             raise ValueError("cycle_rate must be a Rate")
         self._cycle_rate = value
         sleep(0.01)
+
+    @property
+    def clock_source(self) -> int:
+        """Getter/Setter property for clock source selection"""
```

Comment:
The docstring should ideally describe the thing it's affect rather than what it is (a property).  How about:

```suggestion
        """The clock source for the sensor"""
```

This way, whenever someone looks at the documentation online or their IDE shows the description, they intuitively know the thing they're actually changing.

Sorry to be nitpicky!  I really do appreciate all the work you've put in!

Reply:
I know it sounds redundant, but that's okay - to your point about the documentation, at some point if people do need to reference that for more information, that's alright.